specification/monitor/resource-manager/readme.go.md

```diff
@@ -39,6 +39,7 @@ batch:
   - tag: package-2019-11
   - tag: package-2020-10-only
   - tag: package-2021-07
+  - tag: package-2021-09
```

Comment:
There is no definition for this tag, could you add that into the list?

Reply:
sure, please check and see if what I did is ok